Methods.java

```diff
@@ -0,0 +1,39 @@
+package Assignment_3;
+
+public class Methods {
+    public static int sumOfDigits(int number) {

```

Comment:
describe each method in brief through comments including short definition , input parameters and return values .